index.js

```diff
@@ -143,6 +145,11 @@ function RingPop(options) {
 
     this.destroyed = false;
     this.joiner = null;
+
+    this.startTime = Date.now(); //used for calculating uptime
+
+    this.tchannelVersion = require(path.join(path.dirname(require.resolve('tchannel')), 'package.json')).version;
```

Comment:
This'll require some sort of error handling. If the path for some reason does not exist and error will be thrown. Make sure the file exists / exceptions are caught, etc.